pyproject.toml

```diff
@@ -16,17 +16,10 @@ license = {file = "LICENSE"}
 requires-python = ">=3.7"
 dependencies = [
     "apache-airflow >=2.0",
-    "apache-airflow-providers-postgres",
-    "apache-airflow-providers-snowflake",
-    "apache-airflow-providers-google",
     "python-frontmatter",
     "pandas >=1.3.4",
-    "s3fs",
-    "snowflake-sqlalchemy ==1.2.0",
-    "snowflake-connector-python[pandas]",
-    "smart-open[all]>=5.2.1",
     "SQLAlchemy==1.3.24",
-    "sqlalchemy-bigquery==1.3.0"
+    "apache-airflow-providers-postgres",
```

Comment:
If a user needs Postgres support, they’d install `astro[postgres]` and pick up the Postgres provider from the extra, so this shouldn’t be needed.

Reply:
So Postgre is default DB until we have support for sqlite.